src/controllers/trip.controller.js

```diff
@@ -116,6 +116,31 @@ class TripController {
     ResponseService.setSuccess(200, 'List of all available locations', availableLocations);
     return ResponseService.send(res);
   }
+
+  /**
+ *
+ *
+ * @static
+ * @param {req} req
+ * @param {res} res
+ * @returns {response} @memberof Trips
+ */
+  static async editOpenTripRequest(req, res) {
+    const { userData } = req, userId = userData.id, { tripId } = req.params;
+    const findTripRequest = await RequestService.findRequestByProperty({ requesterId: userId }, tripId, { status: !('accepted') });
```

Comment:
The possible value for the status column is either `approved` or `rejected`. What does the `accepted` value do?